### PR TITLE
remove unused rules

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.12/operator_hub.xyaml [hunk 1]
```diff
@@ -35,11 +35,19 @@ create_cr_infinispancluster:
...
+  action: switch_to_yaml_view
   action: submit_changes
+create_operand_anzo:
+  params:
+    operand_kind: Anzo
+  action: create_operand
+create_operand_cluster:
+  params:
+    operand_kind: Cluster
+  action: create_operand
```
Comment (XiyunZhao) [inline lib/rules/web/admin_console/4.12/operator_hub.xyaml:50]: What is the purpose of 'create_operand_anzo' & 'create_operand_cluster'? Seems the changes will not impact cases 44248 & 33716?

Comment (yapei) [reply lib/rules/web/admin_console/4.12/operator_hub.xyaml]: good catch, removed the changes

**Pull request merged**